docs/protocol.html

```diff
@@ -143,7 +138,13 @@ <h5 class="anchor-heading"><a id="api_versions" class="anchor-link"></a><a href=
         Kafka protocol requests may take place on an SSL listener before the SSL handshake is finished). If this is considered to
         leak information about the broker version a workaround is to use SSL with client authentication which is performed at an
         earlier stage of the connection where the <code>ApiVersionRequest</code> is not available. Also, note that broker versions older
-        than 0.10.0.0 do not support this API and will either ignore the request or close connection in response to the request.</li>
+        than 0.10.0.0 do not support this API and will either ignore the request or close connection in response to the request. Also
+        note that if the client <code>ApiVersionsRequest</code> version is unsupported by the broker (client is ahead), and the broker
+        version is 2.5.0 or greater, then the broker will respond with a version 0 <a href="#The_Messages_ApiVersions">ApiVersionsResponse</a>
```

Comment:
Where is the `The_Messages_ApiVersions`? I can't find it anywhere.

Reply:
it's generated, see the current docs for an example https://kafka.apache.org/protocol#The_Messages_ApiVersions

Locally I had to do something like `./gradlew siteDocsTar` and follow the instructions on [kafka-site ](https://github.com/apache/kafka-site?tab=readme-ov-file#how-to-preview-the-latest-documentation-changes-in-kafka-repository)

to get the whole page rendered